enhancements/windows-containers/cluster-logging-windows-worker-nodes.md

```diff
@@ -36,116 +35,87 @@ and pods running in an OpenShift cluster
 
 ### Goals
 
 As part of this enhancement, we plan to do the following:
-* Deploy log collection infrastructure onto Windows nodes
-* Collect node and pod logs
-* Upgrade log collection infrastructure
-* Forward logs and events from nodes and pods to log store
+* Deploy log collection infrastructure onto the Windows nodes.
+* Collect node and pod logs.
+* Upgrade log collection infrastructure.
+* Forward logs and events from nodes and pods to log store.
 
 ### Non-Goals
 
 As part of this enhancement, we do not plan to support:
-* Logging for Windows containers
-  * Microsoft will provide us with the logging framework that enables Windows 
-    containers to redirect logs to standard out from various sources like 
-    ETW, Perf Counters etc
-* Customization of log collection infrastructure
-  * We plan to support fluentd only
-  * Microsoft will provide us with a Windows fluentd binary or gem file
+* Logging for Windows containers.
+* Customization of log collection infrastructure.
+  * We plan to use the Microsoft provided Windows Fluentd binary and the Fluentd
+    logging driver that enables Windows containers to redirect logs to various
+    destinations using the Fluentd output plugins.
 
 
 ## Proposal
 
-The main idea here is to deploy fluentd as a log collection agent onto Windows 
-nodes after cluster logging operator is deployed onto the cluster. Fluentd has
-support for Windows logging. We plan to leverage it and run fluentd as a 
-Windows service.
+The main idea here is to deploy Fluentd as a log collection agent onto Windows 
+nodes after cluster logging operator is deployed onto the OpenShift cluster. 
+Fluentd has support for Windows logging. We plan to leverage it and run Fluentd 
+as a Windows service.
 
 
 ### Implementation Details
 
-We plan to create a Windows Node logging Ansible Playbook that can
-
-* Download fluentd binary for Windows from a known location and transfers it to
-  the Windows node
-* Configure the container runtime to use fluentd as the logging driver
-* Configure fluentd to communicate with elasticsearch data store in the same 
-  format as logs emitted from linux worker nodes
-
-The inputs to this playbook
-
-* The ip address, port, certificate to connect the elasticsearch service
-* URL from which fluentd Windows binary can be downloaded from
-
-We can get the ipaddress and other details from various pods and secrets present
-in the OpenShift cluster, once the cluster logging operator has been deployed.
-If the certificates change, cluster admin should run the ansible playbook
-again. 
-
-Kubelet on Windows nodes logs to a file. So, we also need to configure 
-fluentd's record transformer to ensure that logs collected from the kubelet log
-file are in format understandable to elasticsearch
+Once WMCO is installed from the OperatorHub and Cluster Logging operator is deployed 
+onto the cluster, the WMCO will perform the following steps to configure logging:
```

Comment:
Can be changed to  `WMCO will perform the following steps to configure logging once its installed from OperatorHub and Cluster Logging operator is deployed` to avoid repetition